data/react-entrypoint-generator.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * This is a script that generates static React class files for two 
```

Comment:
This creates the react component based on the congressman data in our JSON file. We might want to bring the react class template into a separate file. Just had it here to make it easy for testing.